com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/validators/parameters/ApplicationNameValidator.java

```diff
@@ -38,11 +41,20 @@ public boolean canCorrect() {
     }
 
     @Override
-    public Object attemptToCorrect(Object applicationName) {
+    public Object attemptToCorrect(Object applicationName, final Map<String, Object> relatedParameters) {
         if (!(applicationName instanceof String)) {
             throw new ContentException(Messages.COULD_NOT_CREATE_VALID_APPLICATION_NAME_FROM_0, applicationName);
         }
-        return NameUtil.computeValidApplicationName((String) applicationName, namespace, useNamespaces);
+
+        Boolean applyNamespaceLocal = NameUtil.parseBooleanFlag(relatedParameters, SupportedParameters.APPLY_NAMESPACE);
+        boolean applyNamespace = NameUtil.resolveApplyNamespaceFlag(applyNamespaceGlobal, applyNamespaceLocal);
```

Comment:
the MapUtil method does some additional parsing ("1" would be considered as true); also resolve method is needed because the global value isn't simply a default e.g.: global values of 'false' and an excplicit local value of 'true' -> this should be resolved to 'false'